images/linux/scripts/tests/PowerShellModules.Tests.ps1

```diff
@@ -0,0 +1,65 @@
+Describe "PowerShellModules" {
+    $modules = (Get-ToolsetContent).powershellModules
+    $withoutVersionsModules = $modules | Where-Object {-not $_.versions} | ForEach-Object {
+        @{moduleName = $_.name}
+    }
+
+    $withVersionsModules = $modules | Where-Object {$_.versions} | ForEach-Object {
+        $moduleName = $_.name
+        $_.versions | ForEach-Object {
+            @{moduleName = $moduleName; expectedVersion = $_}
+        }
+    }
+
+    It "<moduleName> is installed" -TestCases $withoutVersionsModules {
+        Get-Module -Name $moduleName -ListAvailable | Should -BeTrue
+    }
+
+    if ($withVersionsModules) {
+        It "<moduleName> with <expectedVersion> is installed" -TestCases $withVersionsModules {
+            (Get-Module -Name $moduleName -ListAvailable).Version -contains $expectedVersion | Should -BeTrue
+        }
+    }
+}
+
+Describe "AzureModules" {
```

Comment:
Maybe split PoweShell and Azure modules to 2 separate files? These modules seem unrelated to each other

Reply:
I don't think we should split on two files, because it's more obvious to store all tests for PowerShell modules in a single place.